lib/template-mixins.js

```diff
@@ -355,7 +420,7 @@ module.exports = function (fields, options) {
         res.locals.selected = function () {
             return function (txt) {
                 var bits = txt.split('='),
-                    val;
+                val;
```

Comment:
right you are - atom's auto-cleanup tool doesn't respect comma separated vars! will fix